### DISCUSSION TITLE
Angular 10 doesn't works with Swiper 7 and Upper

### DISCUSSION THREAD

**Discussion by carlosminatoro** [nolimits4web/swiper#5402] [Q&A]
It's showing this error when I build or run the app

ERROR: node_modules/swiper/angular/angular/src/swiper-slide.directive.d.ts:23:21 - error TS2694: Namespace '"/Users/xxxxxxxxx/Desktop/xxxxxxxxx/sprint_31/xxxxxxxxxxxxxxxx/node_modules/@angular/core/core"' has no exported member 'ɵɵFactoryDeclaration'.

Any Idea?

**Comment by vltansky** [comment]
can you make codesanbox?
